packages/itwinui-react/src/core/Menu/Menu.tsx

```diff
@@ -243,15 +243,15 @@ export const Menu = React.forwardRef((props, ref) => {
     focusableItemIndex,
     userProps,
   }) => {
-    return popover.getItemProps({
+    return getItemProps({
```

Comment:
No, `popover.getItemProps` does nothing. `useRole` only sets `getItemProps` if `combobox`/`select` roles are used (which we don't use anywhere).

Further, I don't think we should even use `useRole` anywhere. I don't trust floating-ui to handle accessibility correctly. The main purpose of this library is to handle positioning and visibility of floating elements.

Reply:
Sure I can add it, but it gives the false impression that `getItemProps` does something useful.

I'm honestly more inclined to remove it from the return value. The `usePopover` hook should be kept simple because it is used in so many places.

Reply:
Removed in [`646916f`](https://github.com/iTwin/iTwinUI/pull/2117/commits/646916fd198ef864bf3ad074d592cf7187fc5e1f)